src/containers/profiles/EditProfiles/EditProfiles.test.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
+import { createMemoryHistory } from 'history';
+import EditProfile, { mapStateToProps } from './index';
+
+const user = {
+  username: 'bevkololi',
+  image: '',
+  bio: 'This is some bio',
+};
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+const store = mockStore({
+  userProfiles: {
+    profile: { ...user },
+  },
+});
+
+const props = {
+  user,
+  getUserProfile: jest.fn(),
+  editUserProfile: jest.fn(),
+  onChange: jest.fn(),
+};
+
+const history = createMemoryHistory(`profiles/edit/${user.username}`);
+
+const wrapper = mount(<EditProfile {...props} store={store} history={history} />);
+
+
+describe('tests users can edit their profiles', () => {
```

Comment:
@bevkololi The suit could read better if it was something like `The EditProfile container`.